index.bs

```diff
@@ -0,0 +1,588 @@
+<pre class="metadata">
+Title: WebVR
+Status: ED
+ED: https://github.com/mozvr/webvr-spec/
+Shortname: webvr
+Level: 1
+Editor: Vladimir Vukicevic, Mozilla https://mozilla.org/, vladimir@mozilla.com
+Editor: Brandon Jones, Google http://google.com/, bajones@google.com
+Editor: Kearwood Gilbert, Mozilla https://mozilla.org/, kgilbert@mozilla.com
+Editor: Chris Van Wiemeersch, Mozilla https://mozilla.org/, cvan@mozilla.com
+Abstract: This specification describes support for accessing virtual reality devices, including sensors and head-mounted displays on the Web.
+Repository: mozvr/webvr-spec
+Mailing List: web-vr-discuss@mozilla.org
```

Comment:
Do you prefer to have spec discussions via GH issues? That is what most active specs use. If so, you want to point to w3c/webvr/issues here.

We can use use https://github.com/w3c/github-notify-ml-config to sync the W3C mailing list https://lists.w3.org/Archives/Public/public-webvr/ with GH issues.